AnimView/CMakeLists.txt

```diff
@@ -57,7 +57,12 @@ else()
   )
 endif()
 
-target_link_libraries(AnimView RncLib)
+if(NOT RncLib)
```

Comment:
I think you are trying to use the cmake for AnimView in isolation? It's not meant to be used that way. You can cmake the root project with all components and then target individual components with make/ninja.